### PR TITLE
issue #12874 - nls: add missing removeSession localization when signi…

### LINKED ISSUE #12874: nls: add missing `removeSession` localization when signing out
#### Feature Description:

There is a missing signing out localization for the `removeSession` method:

https://github.com/eclipse-theia/theia/blob/b3e8d425155eb92bb8a68773b2b7bb2f9584183b/packages/plugin-ext/src/main/browser/authentication-main.ts#L362-L365

We can benefit from the default localization:

```diff
removeSession(sessionId: string): Thenable<void> {
    return this.proxy.$removeSession(this.id, sessionId)
-       .then(() => { this.messageService.info('Successfully signed out.'); });
+       .then(() => this.messageService.info(nls.localizeByDefault('Successfully signed out.')));
}
```

### PR DESCRIPTION
**Pull request by peter-dimitrov**
#### What it does
Fixes #12874 

#### How to test
Can test by signing out

#### Review checklist

- [x] As an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)
)

### CODE REVIEW

**Review by vince-fugnitto — COMMENTED** [review]
Thank you for your contribution :+1: 

In order to accept your changes please be sure to sure to sign the [eclipse contributor agreement (eca)](https://github.com/eclipse-theia/theia/blob/master/CONTRIBUTING.md#eclipse-contributor-agreement) with the same email as your authorship.

File: packages/plugin-ext/src/main/browser/authentication-main.ts [hunk 1]
```diff
@@ -361,7 +361,7 @@ export class AuthenticationProviderImpl implements AuthenticationProvider {
 
     removeSession(sessionId: string): Thenable<void> {
         return this.proxy.$removeSession(this.id, sessionId)
-            .then(() => { this.messageService.info('Successfully signed out.'); });
+            .then(() => this.messageService.info(nls.localizeByDefault('Successfully signed out.')));
```
Comment (tsmaeder) [inline packages/plugin-ext/src/main/browser/authentication-main.ts:364]: Never mind this comment, I'm just testing something

**Review by tsmaeder — DISMISSED** [review]

**Comment by msujew** [comment]
@peter-dimitrov Please sign the ECA or we won't be able to merge this PR.

**Comment by msujew** [comment]
I'll close this PR if there is no further update in a few days.

**Comment by JonasHelming** [comment]
@msujew I guess we need to close this unfortunately

**Pull request closed without merging**